From 30958578d75b1e1365f3434e9af7662486b1f92e Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Fri, 1 Feb 2013 12:11:24 +0100
Subject: [PATCH 195/628] hwmon: xadcps: Fix coding style

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit d85c1f72c8c16751a698c5de07e12f031840d229

Just solve checkpatch issues.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/hwmon/xilinx-xadcps.c |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/hwmon/xilinx-xadcps.c b/drivers/hwmon/xilinx-xadcps.c
index b2bdf0e..e392981 100644
--- a/drivers/hwmon/xilinx-xadcps.c
+++ b/drivers/hwmon/xilinx-xadcps.c
@@ -267,7 +267,7 @@ static inline void write_register(struct xadc_t *xadc, unsigned int reg,
 	kfree(b);
 }
 
-static inline unsigned reg2temp (u16 reg)
+static inline unsigned reg2temp(u16 reg)
 {
 	unsigned val;
 
@@ -277,7 +277,7 @@ static inline unsigned reg2temp (u16 reg)
 	return val;
 }
 
-static inline unsigned reg2vcc (u16 reg)
+static inline unsigned reg2vcc(u16 reg)
 {
 	unsigned val;
 
@@ -287,7 +287,7 @@ static inline unsigned reg2vcc (u16 reg)
 	return val;
 }
 
-static inline unsigned reg2v (u16 reg)
+static inline unsigned reg2v(u16 reg)
 {
 	unsigned val;
 
@@ -297,7 +297,7 @@ static inline unsigned reg2v (u16 reg)
 	return val;
 }
 
-static inline int reg2bv (u16 reg)
+static inline int reg2bv(u16 reg)
 {
 	int val;
 
@@ -489,7 +489,7 @@ static ssize_t xadc_write_vmode(struct device *dev,
 static ssize_t show_name(struct device *dev,
 		struct device_attribute *devattr, char *buf)
 {
-	return sprintf(buf, "%s\n","xadcps");
+	return sprintf(buf, "%s\n", "xadcps");
 }
 
 static DEVICE_ATTR(name, S_IRUGO, show_name, NULL);
@@ -665,7 +665,7 @@ static int __devinit xadc_probe(struct platform_device *pdev)
 		dev_err(&pdev->dev, "Failed to allocate driver structure\n");
 		return -ENOMEM;
 	}
-	xadc->dev=&pdev->dev;
+	xadc->dev = &pdev->dev;
 
 	xadc->irq = platform_get_irq(pdev, 0);
 	if (xadc->irq < 0) {
-- 
1.7.5.4

